src/Aspire.Hosting.Azure/Provisioning/Provisioners/TokenCredentialHolder.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Azure.Core;
+using Azure.Identity;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Aspire.Hosting.Azure.Provisioning;
+
+internal class TokenCredentialHolder
+{
+    public TokenCredentialHolder(ILogger<TokenCredentialHolder> logger, IOptions<AzureProvisionerOptions> options)
```

Comment:
Do you want to change the category of these logs from `AzureProvisioner` to `TokenCredentialHolder`?

Reply:
Not in this PR 😄

Reply:
I meant this PR is changing the logging category. Is that really what you want?

Reply:
Ah, no I would rather keep it the same as it was.